app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,80 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?   
+	  erb :'tweets/index'
+	else
+	  redirect to '/login'
+	end  
+  end
+
+  get '/tweets/new' do
+  	if logged_in?
+  	  erb :'tweets/new'
+  	else
+  		redirect to '/login'
+  	end  
+  end
+
+  post '/tweets' do
+    @user = User.find_by_id(session[:user_id])
```

Comment:
You can take advantage of your `current_user` helper here. 

`@user = current_user`